lib/LaTeXML/Post/MathML.pm

```diff
@@ -729,15 +731,18 @@ sub stylizeContent {
     $class = ($class ? $class . ' ' : '') . 'ltx_font_oldstyle'; }
   elsif ($font =~ /smallcaps/) {
     $class = ($class ? $class . ' ' : '') . 'ltx_font_smallcaps'; }
+  elsif ($variant) {    # Any left-over mathvariant? Punt to CSS
+    $class = ($class ? $class . ' ' : '') . 'ltx_mathvariant_' . $variant; }
```

Comment:
Should this check also unset `$variant = undef;` after it is moved to a class value?

As it is, I see that one of the tests has both a class and mathvariant attribute, as in:
```xml
<mtext class="ltx_mathvariant_italic" mathvariant="italic">
```

But in MathML Core the only prescribed use of mathvariant is for the `normal` value. So the `class` addition should suffice. That was my suggestion in https://github.com/brucemiller/LaTeXML/issues/2051#issuecomment-1676419924

Reply:
Right, well as I understood the original issue, and the implications of the (eventually) linked mozilla issue, it was that Core will ignore (most) mathvariants, and that Full *may* ignore it, and that CSS could be used as a fallback. Moreover, that browsers should be able to cope with there being both CSS and mathvariant.

Keeping the mathvariant at least records for MathML-Full agents which don't support CSS that the token was (apparently) intended to belong to a certain semantic class.  Removing it seems somehow wrong, even though in the current scheme of things it is likely always to be ignored (but harmless?)

Reply:
Since it is ignored in practice, keeping it until there is some actual issue caused by it is likely harmless, as you say. And indeed it is a MathML Full use.

I've heard rumors of hypothetical MathML Full agents without CSS support, but I wonder if/when we'll have some to test with...

Linking the current MathML Core text on [mathvariant](https://w3c.github.io/mathml-core/#dfn-mathvariant) just in case:
> The mathvariant attribute, if present, must be an [ASCII case-insensitive](https://infra.spec.whatwg.org/#ascii-case-insensitive) match of normal.

---

No further comments on the PR, looks good to merge.